pkg/web/apps/apps_repository.go

```diff
@@ -13,6 +13,11 @@ type Repository interface {
 	DisableAllAppVersionsByAppID(appID string, message string) error
 	DeleteAppByAppID(appId string) error
 	CreateApp(id, appId, name string) error
-	GetAppByAppID(appID string) (*models.App, error)
```

Comment:
Here, it is missing came back to the project this method. Also, the binding service should still be using this one and not the new one GetActiveAppByAppID
@craicoverflow @davidffrench

Reply:
Fixed @camilamacedo86